src/autoscaler/metricscollector/collector/app_streamer.go

```diff
@@ -107,7 +107,7 @@ func (as *appStreamer) processEvent(event *events.Envelope) {
 	} else if event.GetEventType() == events.Envelope_HttpStartStop {
 		as.logger.Debug("process-event-get-httpstartstop-event", lager.Data{"event": event})
 		ss := event.GetHttpStartStop()
-		if ss != nil {
+		if ss != nil && ss.GetPeerType() == events.PeerType_Client {
```

Comment:
Do we need to check whether peertype exists as the nozzle does?

Reply:
No, that is not necessary as it already the default: https://github.com/cloudfoundry/sonde-go/blob/04fbe57ef6878d0050682b5b0697091003ebf960/events/http.pb.go#L262-L267